rust_src/src/remacs_sys.rs

```diff
@@ -75,6 +75,7 @@ pub enum BoolVectorOp {
 extern "C" {
     // these weren't declared in a header, for example
     pub static Vprocess_alist: Lisp_Object;
+    pub static Vstandard_syntax_table: Lisp_Object;
```

Comment:
This is wrong. There's a `syntax_table` field on every `buffer`, and `Vstandard_syntax_table` is a macro that refers to the `syntax_table` field on a particular `buffer` struct which was set aside for the purpose of holding the default values for this and other fields. See https://github.com/Wilfred/remacs/blob/master/src/buffer.h#L1086-L1093

Reply:
https://github.com/Wilfred/remacs/blob/cdf7a08ef6a76a220661b9e6223d547780b825b6/src/syntax.h#L34

You could write a Rust macro to keep the idea.